### ISSUE TITLE
Testing code

### ISSUE THREAD

**Issue by XiaRho** [uzh-rpg/ess#2]
Hi, after downloading the pre-trained weights, how to test your models to generate the semantic segmentation results and get the MIoU index?

**Comment by SunZNing** [comment]
Hi,
Please look at the newly added instructions for testing pre-trained models in the readme.

**Comment by XiaRho** [comment]
Thank you very much. One more question, when I try to reproduce the results (events+frames MIoU=53.29) on DSEC-Semantic in the supervised setting. I have already set the train_on_event_labels to True in settings_DSEC.yaml. However, the final MIoU is about 51.5. This seems to be the results of just using event labels. Could you also provide the pre-trained weights (events+frames MIoU=53.29) on DSEC-Semantic in the supervised setting?
![image](https://user-images.githubusercontent.com/74460633/189319393-32c577d0-b09a-48e0-b539-fa0fc7af28ee.png)

**Comment by ShristiDasBiswas** [comment]
> Thank you very much. One more question, when I try to reproduce the results (events+frames MIoU=53.29) on DSEC-Semantic in the supervised setting. I have already set the train_on_event_labels to True in settings_DSEC.yaml. However, the final MIoU is about 51.5. This seems to be the results of just using event labels. Could you also provide the pre-trained weights (events+frames MIoU=53.29) on DSEC-Semantic in the supervised setting? ![image](https://user-images.githubusercontent.com/74460633/189319393-32c577d0-b09a-48e0-b539-fa0fc7af28ee.png)

Where are you getting the e2vid/pretrained/E2VID_lightweight.pth.tar file as required in the settings.py file?

**Comment by ShristiDasBiswas** [comment]
> Thank you very much. One more question, when I try to reproduce the results (events+frames MIoU=53.29) on DSEC-Semantic in the supervised setting. I have already set the train_on_event_labels to True in settings_DSEC.yaml. However, the final MIoU is about 51.5. This seems to be the results of just using event labels. Could you also provide the pre-trained weights (events+frames MIoU=53.29) on DSEC-Semantic in the supervised setting? ![image](https://user-images.githubusercontent.com/74460633/189319393-32c577d0-b09a-48e0-b539-fa0fc7af28ee.png)

Hi, what are the total trainable parameters that ESS uses? I am unable to generate to number.

**Comment by XiaRho** [comment]
> > Thank you very much. One more question, when I try to reproduce the results (events+frames MIoU=53.29) on DSEC-Semantic in the supervised setting. I have already set the train_on_event_labels to True in settings_DSEC.yaml. However, the final MIoU is about 51.5. This seems to be the results of just using event labels. Could you also provide the pre-trained weights (events+frames MIoU=53.29) on DSEC-Semantic in the supervised setting? ![image](https://user-images.githubusercontent.com/74460633/189319393-32c577d0-b09a-48e0-b539-fa0fc7af28ee.png)
> 
> Hi, what are the total trainable parameters that ESS uses? I am unable to generate to number.

I only set the **model_name** to **ess_supervised** (line 30), **train_on_event_labels** to **True** (line 35) in the setting_DSEC.yaml, and add this code **self.use_task_a = True** to the setting.py. I get the E2VID_lightweight from https://github.com/uzh-rpg/rpg_e2vid.

**Comment by ldkong1205** [comment]
Hi @XiaRho, how much GPU memory is needed to run the experiment? What type of machines are you using? Thanks!

**Comment by XiaRho** [comment]
> Hi @XiaRho, how much GPU memory is needed to run the experiment? What type of machines are you using? Thanks!

About 8000MB memory is need. l use the Tesla V100-SXM2-16GB.

**Comment by ldkong1205** [comment]
> > Hi @XiaRho, how much GPU memory is needed to run the experiment? What type of machines are you using? Thanks!
> 
> About 8000MB memory is need. l use the Tesla V100-SXM2-16GB.

Thank you for your information!

**Comment by MessikommerNico** [comment]
> Thank you very much. One more question, when I try to reproduce the results (events+frames MIoU=53.29) on DSEC-Semantic in the supervised setting. I have already set the train_on_event_labels to True in settings_DSEC.yaml. However, the final MIoU is about 51.5. This seems to be the results of just using event labels. Could you also provide the pre-trained weights (events+frames MIoU=53.29) on DSEC-Semantic in the supervised setting? ![image](https://user-images.githubusercontent.com/74460633/189319393-32c577d0-b09a-48e0-b539-fa0fc7af28ee.png)

Hi
We added the links for the pre-trained weights obtained in the supervised settings to the [Google form](https://docs.google.com/forms/d/e/1FAIpQLScn5XWvBcmjPoSbaIqEUoEpWeheLGpQTUeK6Pp19wx2jNCPpA/viewform?usp=sf_link).

**Comment by XiaRho** [comment]
> > Thank you very much. One more question, when I try to reproduce the results (events+frames MIoU=53.29) on DSEC-Semantic in the supervised setting. I have already set the train_on_event_labels to True in settings_DSEC.yaml. However, the final MIoU is about 51.5. This seems to be the results of just using event labels. Could you also provide the pre-trained weights (events+frames MIoU=53.29) on DSEC-Semantic in the supervised setting? ![image](https://user-images.githubusercontent.com/74460633/189319393-32c577d0-b09a-48e0-b539-fa0fc7af28ee.png)
> 
> Hi We added the links for the pre-trained weights obtained in the supervised settings to the [Google form](https://docs.google.com/forms/d/e/1FAIpQLScn5XWvBcmjPoSbaIqEUoEpWeheLGpQTUeK6Pp19wx2jNCPpA/viewform?usp=sf_link).

Thank you very much!

**Comment by SchuckLee** [comment]
Hello, I would like to know how to set up supervised learning on ess using only events? I have set train_on_event_labels: True and model_name: 'ess_supervised', but I still use frame for training...Thanks a lot!
